.github/workflows/runAqaArgParse.py

```diff
@@ -0,0 +1,78 @@
+import argparse
+import json
+import sys
+
+def map_platforms(platforms):
+    """ Takes in a list of platforms and translates Grinder platorms to corresponding GitHub-hosted runners.
+        This function both modifies and returns the 'platforms' argument.
+    """
+    
+    platform_map = {
+        'x86-64_windows': 'windows-latest',
+        'x86-64_mac': 'macos-latest',
+        'x86-64_linux': 'ubuntu-latest'
+    }
+    
+    for i, platform in enumerate(platforms):
+        if platform in platform_map:
+            platforms[i] = platform_map[platform]
+    
+    return platforms
+
+def underscore_targets(targets):
+    """ Takes in a list of targets and prefixes them with an underscore if they do not have one.
+    """
+    result = []
+    for target in targets:
+        t = target
+        if target[0] != '_':
+            t = '_' + t
+        result.append(t)
+    return result
+
+def main():
+
+    # The keyword for this command.
+    keyword = 'run aqa'
+    keywords = keyword.split()
+
+    # We assume that the first argument is/are the keyword(s).
+    # e.g. sys.argv == ['action_argparse.py', 'run', 'aqa', ...]
+    raw_args = sys.argv[1 + len(keywords):]
+
+    # Strip leading and trailing whitespace. Remove empty arguments that may result after stripping.
+    raw_args = list(filter(lambda empty: empty, map(lambda s: s.strip(), raw_args)))
+
+    parser = argparse.ArgumentParser(prog=keyword, add_help=False)
+    # Improvement: Automatically resolve the valid choices for each argument populate them below, rather than hard-coding choices.
+    parser.add_argument('--sdk_resource', default=['nightly'], choices=['nightly', 'releases', 'customized'], nargs='+')
+    parser.add_argument('--customized_sdk_url', default=['None'], nargs='+')
+    parser.add_argument('--archive_extension', default=['.tar'], choices=['.zip', '.tar', '.7z'], nargs='+')
+    parser.add_argument('--build_list', default=['openjdk'], choices=['openjdk', 'functional', 'system', 'perf', 'external'], nargs='+')
```

Comment:
I saw this in openjdk-tests repo, but not tried yet to find out what happens when someone tries to build external?  Is docker installed on github runners and this works smoothly?

Reply:
I've tried with jenkin_tests and camel tests and hit the same issue:
https://github.com/sophia-guo/runaqaTest/runs/2249338597?check_suite_focus=true
https://github.com/sophia-guo/runaqaTest/runs/2249127830?check_suite_focus=true

Look docker is fine. There are some hidden issues related with environment setup? Both jobs can successfully create docker images but some how `unable to recognize DOCKER_OS` hence docker run is trying to delete default images and find the image is not there. I will take a look.

```
Unable to recognize DOCKER_OS from DOCKERIMAGE_TAG = --version!
docker run  --rm adoptopenjdk-derby-test:8-jdk-ubuntu-hotspot-full 
Unable to find image 'adoptopenjdk-derby-test:8-jdk-ubuntu-hotspot-full' locally

```